index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Backgrounds and Gradients Project</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <script src="https://kit.fontawesome.com/e252d07dd4.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <header id="bg-container">
+      <nav>
+        <ul>
+          <li>
+            <a id="apple" href="#"> <i class="fab fa-apple"></i> </a>
+          </li>
+          <li><a href="#">Store</a></li>
+          <li><a href="#">Mac</a></li>
+          <li><a href="#">iPod</a></li>
+          <li><a href="#">iPhone</a></li>
+          <li><a href="#">iPad</a></li>
+          <li><a href="#">iTunes</a></li>
+          <li><a href="#">Support</a></li>
+          <li class="search-bar">
+            <i class="fas fa-search fa-xs white"></i>
+            <input type="text" />
+          </li>
+        </ul>
+      </nav>
+      <div class="main">
+        <a href="#"><h1>What will you verse be?</h1></a>
+        <a href="#"><h2>Everyone has something to share. See how people are using iPad in their unique ways to add to the world's story.</h2></a>
+        <h3>
+          <a href="#"> Watch the "Your Verse" film and explore the stories</a>
+          <i class="far fa-play-circle" aria-hidden="true"></i>
+        </h3>
+      </div>
+    </header>
+    <div class="middle-container">
+      <section id="images">
+        <img src="images/pic-1.jpg" alt="Image-1" />
+        <img src="images/pic-2.jpg" alt="Image-1" />
+        <img src="images/pic-3.jpg" alt="Image-1" />
+        <img src="images/pic-4.jpg" alt="Image-1" />
+      </section>
+    </div>
+    <div class="middle-container">
+      <footer>
+        <div id="left-footer">
+          <ul>
+            <li>
+              <p>Shop the <a href="#">Apple Online Store</a> (1-800-MY-APPLE), visit an <a href="#">Apple Retail Store</a>, or find a <a href="#">reseller</a>.</p>
+            </li>
+            <li>
+              <p>
+                Copyright © 2014 Apple Inc. All rights reserved.
+                <a href="#">Terms of Use</a>
+                <a href="#">| Privacy Policy</a>
+              </p>
+            </li>
+          </ul>
+        </div>
+        <div id="right-footer">
+          <ul>
+            <li><a href="#">Site Map</a></li>
+            <li><a href="#">Hot News</a></li>
+            <li><a href="#">RSS Feeds</a></li>
+            <li><a href="#">Media Info</a></li>
+            <li><a href="#">Environment</a></li>
+            <li><a href="#">Job Opportunities</a></li>
+            <li><a href="#">Contact Us</a></li>
+            <li>
+              <a href="#"><img src="images/usa.png" alt="usa" /></a>
+            </li>
+          </ul>
+        </div>
+
+        <!-- https://web.archive.org/web/20140301004353im_/http://images.apple.com/global/elements/flags/16x16/usa.png -->
```

Comment:
* [ ] Comments: It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
  :x: **commented piece of codes** /\* `.nav { color: red;}` \*/  
  :heavy_check_mark: **comments about code** /\*Navigation \*/